doc/Comments.xml

```diff
@@ -300,9 +300,11 @@ Resets the current level to 0.
 <Subsection Label="@BeginExample">
 <Index Key="@BeginExample"><C>@BeginExample and @EndExample</C></Index>
 <Heading>@BeginExample and @EndExample</Heading>
-@BeginExample inserts an example into the manual. The syntax is like the example environment
```

Comment:
Typo in commit message: "documtentation"

Reply:
Also, this doesn't just "fix" the documentation... Perhaps: "Improve documentation for @Example"